frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -706,6 +725,7 @@ pub mod pallet {
 		pub fn submit_unsigned(
 			origin: OriginFor<T>,
 			solution: RawSolution<CompactOf<T>>,
+			round: u32,
```

Comment:
Note that we should add the same in #7910 as well.

Reply:
We actually shouldn't; see https://github.com/paritytech/substrate/pull/8290#discussion_r598685772